test/jdk/valhalla/valuetypes/StaticFactoryTest.java

```diff
@@ -24,8 +24,9 @@
 
 /*
  * @test
+ * @bug 8273360
  * @summary Test reflection of constructors for primitive classes
- * @run testng/othervm StaticFactoryTest
+ * @run testng/othervm -Dsun.reflect.noInflation=true StaticFactoryTest
```

Comment:
```suggestion
 * @run testng/othervm StaticFactoryTest
 * @run testng/othervm -Dsun.reflect.noInflation=true StaticFactoryTest
```

This will exercise both code paths (with and without the spinned class).